pkg/cmd/admin/router/router.go

```diff
@@ -49,14 +49,14 @@ you have failover protection.`
 	routerExample = `  // Check the default router ("router")
   $ %[1]s %[2]s --dry-run
 
-  // See what the router would look like if created
-  $ %[1]s %[2]s -o json
+  // See what the router will look like if created
+  $ %[1]s %[2]s -o json --credentials=router-user.kubeconfig --service-account=router
```

Comment:
router commands were updated in https://github.com/openshift/origin/pull/4512.  Looks like the registry commands could still use an update.